frontend/src/app/components/Footer/index.scss

```diff
@@ -7,7 +9,10 @@
     padding: $grid-unit * 1 0;
   }
 
-  background: $transparent-white-1;
+  .newsletter-footer + & {
```

Comment:
This is actually the whole PR, everything else is just refactoring for components